misc/testerina/modules/testerina-core/src/main/java/org/ballerinalang/testerina/core/TestAnnotationProcessor.java

```diff
@@ -540,4 +543,12 @@ private String getPackageName(PackageNode packageNode) {
         BLangPackage bLangPackage = ((BLangPackage) packageNode);
         return bLangPackage.packageID.toString();
     }
+
+    private static int getInstructionsEndLimit(PackageInfo packageInfo) {
```

Comment:
Better to add some explanation.

Reply:
Since this is a private function I am bit reluctant to add explanations. As per my belief most of the time just the name of the function should be expressible enough. WDYT? Renamed the function name  in https://github.com/ballerina-platform/ballerina-lang/pull/13418/commits/8901a7c96c5aaba4d0958db616d4f4333c19be3e.